react/ProductSummaryContext.tsx

```diff
@@ -115,11 +115,11 @@ const buildProductQuery = ((product: Product) => {
   return querystring.stringify(query)
 })
 
-function ProductSummaryProvider({ product, selectedItem, children }) {
+function ProductSummaryProvider({ product, selectedItem, isLoading, children }) {
```

Comment:
Adding a default value just to be backwards compatible

```suggestion
function ProductSummaryProvider({ product, selectedItem, isLoading = false, children }) {
```

Reply:
Done!